anndata/_io/specs/methods.py

```diff
@@ -303,10 +306,30 @@ def write_list(f, k, elem, dataset_kwargs=MappingProxyType({})):
 @_REGISTRY.register_write(ZarrGroup, h5py.Dataset, IOSpec("array", "0.2.0"))
 @_REGISTRY.register_write(ZarrGroup, np.ma.MaskedArray, IOSpec("array", "0.2.0"))
 def write_basic(f, k, elem, dataset_kwargs=MappingProxyType({})):
-    """Write methods which underlying library handles nativley."""
+    """Write methods which underlying library handles natively."""
     f.create_dataset(k, data=elem, **dataset_kwargs)
 
 
+@_REGISTRY.register_write(ZarrGroup, da.Array, IOSpec("dask-array", "0.2.0"))
+def write_zarr(f, k, elem, dataset_kwargs=MappingProxyType({})):
+    g: ZarrGroup = f.create_group(k)
```

Comment:
You shouldn't need to create a group here. As for  the other dense arrays, this can be saved directly to the key.